File: games/planetization/bots/java/src/core/api/UnitData.java
package core.api;

import core.api.commands.UnitType;

public class UnitData {
    public int id;
    public UnitType type;
    public float x, y;
    public float rotation;
    public float health;
    public int currentPlanetId;
    public int destinationPlanetId;
}

File: games/planetization/bots/kotlin/src/core/api/UnitData.kt
package core.api

class UnitData(
        var id: Int,
        var type: UnitType,
        var x: Float,
        var y: Float,
        var rotation: Float,
        var health: Float,
        var currentPlanetId: Int,
        var destinationPlanetId: Int
)
